gwardia/templates/gwardia/index.html

```diff
@@ -3,5 +3,40 @@
 {% block title %}Panel Gwardii{% endblock %}
 
 {% block content %}
-    <h1>Panel Gwardii! WIP!</h1>
+    <div class="flex flex-col items-center h-screen">
+        <h1 class="text-4xl text-white font-bold my-10 mt-16 mb-8">Panel Gwardii! WIP!</h1>
+        <p class="italic text-xs text-gray-500 mb-2">Click meeting to display its description</p>
+        <ul class="w-1/2" id="js-next-meetings-container">
+            {% for meeting in next_5_meetings %}
+                <li class="js-meeting-container flex flex-col items-center">
+                    <p class="border-solid border-gray-500 border-2 p-5 w-full mb-1 rounded-2xl flex justify-between">
```

Comment:
Noooo! Flex again! Maybe remove it (and `justify-between`) and add `float-right` to `date_of_meeting`

Reply:
Kys